themes/openy_themes/openy_carnation/openy_carnation.libraries.yml

```diff
@@ -2,7 +2,7 @@ global-styling:
   version: 1.x
   css:
     base:
-      //use.fontawesome.com/releases/v5.1.0/css/all.css: {type: external, attributes: { defer: true, async: true} }
+      //use.fontawesome.com/releases/v5.1.0/css/all.css: {type: external}
```

Comment:
why? this was performance tuning

Reply:
defer and async don't work for css, only for scripts. 
first request with redirect https://i.imgur.com/ORHixFG.png
second https://i.imgur.com/uLvXcmx.png

https://www.w3schools.com/tags/att_defer.asp
https://www.w3schools.com/tags/att_async.asp